ecmd-core/ext/fapi2/capi/plat_target_rules.H

```diff
@@ -852,6 +852,16 @@ constexpr void _check_valid_parent_of_iohs()
                   "improper parent of fapi2::TARGET_TYPE_IOHS");
 }
 
+template< fapi2::TargetType K, fapi2::TargetType T >
```

Comment:
Irrelevant to this change, but just a future suggestion -- If K/T were CHILD/PARENT it makes it easier to read :-)